lib/components/composed_component.dart

```diff
@@ -70,6 +75,10 @@ mixin ComposedComponent on Component, HasGameRef, Tapable {
     components.add(c);
   }
 
+  void remove(Component component) {
+    _removeLater.add(component);
+  }
+
   // this is an important override for the Tapable mixin
   @override
```

Comment:
@luanpotter heads up on the positionComponent refactor due to these changes on composed.

Reply:
yep I will need to integrate this on the new structure